packages/shared-components/src/components/onto-header/onto-header.tsx

```diff
@@ -7,13 +8,32 @@ import { Component, Host, h } from '@stencil/core';
 })
 export class OntoHeader {
 
+  private languageService: LanguageService;
+
+  // TODO remove this when implement language selector
+  private locale = 'en'
+
+  constructor() {
+    this.languageService = ServiceProvider.get(LanguageService);
+  }
+
+  // TODO remove this when implement language selector
+  private onLanguageChanged(): void {
+    if (this.locale === 'en') {
+      this.locale = 'fr';
+    } else {
+      this.locale = 'en';
+    }
+    this.languageService.changeLanguage(this.locale);
+  }
+
   render() {
     return (
       <Host>
         <div class="header-component">
           <div class="search-component">&#x1F50D;</div>
           <div class="repository-selector-component">TestRepo &#8964;</div>
-          <onto-language-selector></onto-language-selector>
+          <div class="language-selector-component" onClick={this.onLanguageChanged}>EN &#8964;</div>
```

Comment:
hmm you need to rebase the branch because the language selector is already in place